src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -224,6 +228,10 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
             CourseAccessStorageService.STORAGE_KEY_DROPDOWN,
             CourseAccessStorageService.MAX_DISPLAYED_RECENTLY_ACCESSED_COURSES_DROPDOWN,
         );
+        this.printSubscription = this.printService.isNavbarCollapsed$.subscribe((isCollapsed) => {
+            this.isNavbarCollapsed = isCollapsed;
+            this.changeDetectorRef.detectChanges();
+        });
```

Comment:
Why do we need this?

Isn't this covered with the CSS changes?

```
@media print {
    .sidebar-container {
        width: $menu-width-closed !important;
    }

    .mat-drawer {
        width: $menu-width-closed !important;
    }

    .mat-drawer-content {
        width: calc(100vw - $menu-width-closed) !important;
        margin-left: $menu-width-closed !important;
    }
}
```

Reply:
only css didnt work out but i removed the css approach now and did it with the subscription 👍